Library/Formula/sqlite-analyzer.rb

```diff
@@ -0,0 +1,18 @@
+class SqliteAnalyzer < Formula
+  homepage "https://www.sqlite.org/"
+  url "https://www.sqlite.org/2015/sqlite-src-3080803.zip"
```

Comment:
I considered this and decided against it, see my initial notes above from when I opened the pull request. The sqlite project also distributes the analyzer separately.